cloudformationTemplates/startChatContactAPI/README.md

```diff
@@ -346,3 +346,21 @@ If you want to enable sending attachments for Amazon Connect Chat the customer c
         }
     },successHandler, failureHandler)
 ```
+
+## Enabling message receipts
+
+To render/send message receipts, enable the feature [in the admin console](https://docs.aws.amazon.com/connect/latest/adminguide/message-receipts.html), and update [`chat-interface`](https://github.com/amazon-connect/amazon-connect-chat-interface#message-receipts) to generate latest webpack bundle.
+
+Ensure that the ChatJs feature configuration is set as well:
```

Comment:
same comment as above; this should be part of `disabling` not `enabling` section.